.github/workflows/agw-workflow.yml

```diff
@@ -5,18 +5,28 @@ on:  # yamllint disable-line rule:truthy
   push:
     branches:
       - master
-    paths:
-      - 'orc8r/**'
-      - 'lte/**'
   pull_request:
     branches:
       - master
       - 'v1.*'
-    paths:
-      - 'orc8r/**'
-      - 'lte/**'
+    types: [opened, reopened, synchronize]
+
 jobs:
+  path_filter:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          # Only run the main job for changes including the following paths
+          paths: '["orc8r/**", "lte/**"]'
   lte-test:
+    needs: path_filter
+    if: ${{ needs.path_filter.outputs.should_skip != 'true' }}
```

Comment:
here too

Reply:
@quentinDERORY - Addressed the comments and pushed the changes